packages/ppx_deriving_json_schema/ppx_deriving_json_schema.1.0.0/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "Ppx deriver for generating JSON schema objects from type definitions"
+maintainer: ["J. Aaron Pendergrass <james.pendergrass@jhuapl.edu>"]
+authors: ["J. Aaron Pendergrass <james.pendergrass@jhuapl.edu>"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/jhuapl-saralab/openapi-ocaml"
+bug-reports: "https://github.com/jhuapl-saralab/openapi-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ppxlib" {>= "0.31.0"}
+  "core" {>= "v0.16.0"}
+  "openapi" {= "1.0.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
    "@runtest" {with-test}
```
We should either disable the tests (which I would not recommend) or make https://github.com/jhuapl-saralab/openapi-ocaml/blob/785a0be72fa4cad4ce3a858a082b5b741a2d0f13/test/dune#L57C1-L60C64 optional and only called in your CI since check-jsonschema is not generally available and will then break the tests on the opam-repository CI

Reply:
Agreed, thanks for pointing that out. Will make that optional.